include/grpc/event_engine/event_engine.h

```diff
@@ -517,23 +518,18 @@ void SetDefaultEventEngine(std::shared_ptr<EventEngine> engine);
 /// EventEngine instance.
 std::shared_ptr<EventEngine> GetDefaultEventEngine();
 
-/// Waits for all refs on the DefaultEventEngine to be released, and resets the
-/// CreateEventEngine factory to create and return one of the default internal
-/// EventEngines.
+/// Resets the CreateEventEngine factory to create and return one of the default
```

Comment:
I don't think this affects `CreateEventEngine()`; I think it affects only `GetDefaultEventEngine()`.

Also, we should not refer to factories in this documentation, since that's not something we're going to be exposing from this API anymore.

Reply:
I reworded it a bit. This affects both Create & Get functions, since `Get` relies on `Create` under the hood.

Note that `CreateEventEngine` isn't going away, and that function is a factory. The user-provided factories are going away, but that's unrelated. Calling it a factory conveys good information.

Reply:
Ah beans, I was mistaken. Documentation fixed.